ci/deps/azure-37-locale.yaml

```diff
@@ -1,5 +1,6 @@
 name: pandas-dev
 channels:
+  - defaults
```

Comment:
hmm why are you adding this?

Reply:
I think this one was originally azure-36-locale.yaml. I renamed it. Looks like GitHub didn't get this right. If you look at the side by side comparison for "ci/deps/azure-36-locale.yaml → ci/deps/azure-38-locale.yaml" below, the "-defaults" channel was there.